AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Models/DynamicModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+struct DynamicModel: JSONModel {
```

Comment:
1. As noted above, let's make sure this conforms to both `Model` and `JSONAccessible` (or whatever)
1. As discussed, please document this as an example, although I should have caught this by the fact that it was in the Tests package.

Reply:
1. Changed
2. Will add